schema/annotation-source.yaml

```diff
@@ -73,12 +73,12 @@ $defs:
         const: Contribution
         default: Contribution
         description: MUST be "Contribution".
-      agent:
+      contributor:
```

Comment:
I will defer to @larrybabb's judgement on this attribute name.

Reply:
I'm cool with this.